src/Coccinelle-for-Rust.md

```diff
@@ -43,6 +43,11 @@ flow is not yet supported.
 
 [A recent talk about Coccinelle for Rust](https://gitlab.inria.fr/coccinelle/coccinelleforrust/-/blob/main/talks/rfl.pdf)
 
+## Support
```

Comment:
In the commit title you mentioned "Sponsors" -- not sure if that word should be used somewhere here if that is the case.

Reply:
I am sorry, I should not have used that word. Thank you!